Apromore-Custom-Plugins/Process-Simulation-Info-Logic/src/main/java/org/apromore/processsimulation/service/SimulationInfoService.java

```diff
@@ -224,6 +230,34 @@ private void deriveGatewayProbabilities(
         }
     }
 
+    private TimeUnit getDisplayTimeUnit(double timeDurationMillis) {
+        TimeUnit timeUnit;
+
+        if (timeDurationMillis <= 60000) {
+            timeUnit = TimeUnit.SECONDS;
+        } else if (timeDurationMillis > 60000 && timeDurationMillis <= 3600000) {
+            timeUnit = TimeUnit.MINUTES;
+        } else if (timeDurationMillis > 3600000) {
+            timeUnit = TimeUnit.HOURS;
+        } else {
+            timeUnit = TimeUnit.valueOf(config.getDefaultTimeUnit().toUpperCase(DOCUMENT_LOCALE));
+        }
```

Comment:
I feel like this could be simplified to just:
```
timeUnit = timeDurationMillis <= 60000   ? TimeUnit.SECONDS :
           timeDurationMillis <= 3600000 ? TimeUnit.MINUTES :
                                           TimeUnit.HOURS;
```

Reply:
I'm not a huge fan of using the ternary operator, as it reduces readability. I typically use it sparingly, where there's only 2 branching possibilities

Reply:
Just as long as you've considered collapsing that logic and whether that final unreachable branch deserves to continue existing, that'll all I was really interested in.